base/range.jl

```diff
@@ -47,25 +47,15 @@ function _colon(start::T, step, stop::T) where T
 end
 
 """
-    range(start[, stop]; length, stop, step=1)
+    range(start[, stop]; length, stop, step)
+    range(;start, length, stop, step)
```

Comment:
There is no method for `range(1, 5; stop=4)` so this is a little bit misleading.

```suggestion
    range(start, stop; length, step)
    range(start; length, stop, step)
    range(;start, length, stop, step)
```